doc/tutorial/text_analytics/solutions/exercise_02_sentiment.py

```diff
@@ -55,7 +55,11 @@
 
     # TASK: print the cross-validated scores for the each parameters set
     # explored by the grid search
-    print(grid_search.grid_scores_)
+    n_candidates = len(grid_search.candidate_params_)
+    test_split_keys = filter(lambda key: key.startswith('test_split_'),
```

Comment:
You should extract the means and stds using
`mean_score = grid_search.results_["test_%s_mean" % gscv.scorer_name_]` instead of this not-so-beautiful hack here.